infrastructure/fluentbit/frontend-fluentbit-config.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: frontend-fluentbit-config
+  namespace: ${NAMESPACE}
+  labels: 
+    app.kubernetes.io/part-of: ${OC_ENV}
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+      Flush         5
+      Daemon        Off
+      # define the log format (see additional config map key/value)
+      Parsers_File  parsers.conf
+      Log_Level     info
+      HTTP_Server   On
+      HTTP_Listen   0.0.0.0
+      HTTP_Port     2020
+      Health_Check  On
+
+    [INPUT]
+      # get logs from file written by nginx app
+      Name        tail
+      Path        /tmp/error_fluentbit.log
+      Tag         app
+  
+    [FILTER]
+      # filter logs based on certain keyword(s)
+      name   grep
+      match  app
+      regex  log error
+    
+    [FILTER]
+      # modify log entry to include namespace and container name
+      name    record_modifier
+      match   app
+      # add namespace
+      Record namespace ${NAMESPACE}
+      # add container name
+      Record product ${OC_ENV}-frontend
+
+    [FILTER]
+      name          parser
+      match         app
+      Key_Name      log
+      Parser        json
+      Reserve_Data  On
+      Preserve_Key  On
+
+    [FILTER]
+      Name          rewrite_tag
+      Match         app
+      Rule          $level ([a-zA-Z]*)$ $TAG.$level true
+      Emitter_Name  re_emitted
+
+
+    [OUTPUT]
+      name                 slack
+      match                app
+      webhook              ${SLACK_WEBHOOK}
+
+    [OUTPUT]
+      Name    stdout
+      Match   app
+      Format  json_lines
+
+  parsers.conf: |
+    [PARSER]
+      Name             json
+      Format           json
```

Comment:
Might be more appropriate to use https://docs.fluentbit.io/manual/pipeline/parsers/regular-expression